include/picongpu/plugins/radiation/amplitude.hpp

```diff
@@ -33,14 +34,16 @@ namespace picongpu
         {
             /** class to store 3 complex numbers for the radiated amplitude
              */
+            template<typename base_dtype = picongpu::float_64>
```

Comment:
We normally call template argument types following `TType` notation

Reply:
@sbastrakov is right: https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#16-template-declarations-and-specializations

Reply:
Renamed `base_dtype` to `T_Float` in 539449d .